firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalStore.java

```diff
@@ -329,50 +329,35 @@ public SnapshotVersion getLastRemoteSnapshotVersion() {
     return persistence.runTransaction(
         "Apply remote event",
         () -> {
+          SnapshotVersion remoteVersion = remoteEvent.getSnapshotVersion();
+          Map<Integer, TargetChange> targetChanges = remoteEvent.getTargetChanges();
           long sequenceNumber = persistence.getReferenceDelegate().getCurrentSequenceNumber();
-          Set<DocumentKey> authoritativeUpdates = new HashSet<>();
 
-          Map<Integer, TargetChange> targetChanges = remoteEvent.getTargetChanges();
           for (Map.Entry<Integer, TargetChange> entry : targetChanges.entrySet()) {
             Integer boxedTargetId = entry.getKey();
             int targetId = boxedTargetId;
             TargetChange change = entry.getValue();
 
-            // Do not ref/unref unassigned targetIds - it may lead to leaks.
```

Comment:
Does this comment no longer apply?

Reply:
I rewrote it below since I didn't quite understand it at first.

Reply:
Ah, cool. I also found the original comment confusing.